easybuild/easyconfigs/f/freetype/freetype-2.5.5-intel-2015b.eb

```diff
@@ -0,0 +1,21 @@
+name = 'freetype'
+version = '2.5.5'
+
+homepage = 'http://freetype.org'
+description = """FreeType 2 is a software font engine that is designed to be small, efficient, highly customizable, and
+ portable while capable of producing high-quality output (glyph images). It can be used in graphics libraries, display
+ servers, font conversion tools, text image generation tools, and many other products as well."""
+
+toolchain = {'name': 'intel', 'version': '2015b'}
+
+source_urls = [GNU_SAVANNAH_SOURCE]
+sources = [SOURCE_TAR_GZ]
+
+dependencies = [('libpng', '1.6.16')]
```

Comment:
well, not older (it's the most recent matplotlib), it just hasn't gotten an update in a while (so this is just a toolchain update w.r.t. matplotlib)